.travis.yml

```diff
@@ -8,8 +8,8 @@ notifications:
   email: false
 before_script:
 - npm prune
-- lerna bootstrap --scope '@(function-tree|cerebral|cerebral-forms|cerebral-provider-http|cerebral-provider-firebase|cerebral-provider-forms|cerebral-router|babel-*)'
-- lerna bootstrap --scope @cerebral/codemods*
+- npm run setup:babel
+- lerna bootstrap --scope cerebral-codemods*
```

Comment:
@Guria Not only to simplify. Something like
```sh
lerna bootstrap --scope '@(function-tree|cerebral|@cerebral/forms'
```
or
```sh
lerna bootstrap --scope '@(function-tree|cerebral|@cerebral/*'
```
doesn't work for me.
See discord release channel 07.05.2017 for detailed discussion.